app/services/DataStoreService.scala

```diff
@@ -37,9 +38,8 @@ class DataStoreService @Inject() (httpClient: HttpClientV2, metricsReporter: Met
 
   val log: Logger = Logger(this.getClass)
 
-  def getAllEoriHistory(eori: EORI)(implicit hc: HeaderCarrier): Future[Seq[EoriHistory]] = {
-    val dataStoreEndpoint = s"${appConfig.customsDataStore}/eori/$eori/eori-history"
-    val emptyEoriHistory  = Seq(EoriHistory(eori, None, None))
+  def getAllEoriHistory()(implicit hc: HeaderCarrier): Future[Seq[EoriHistory]] = {
```

Comment:
Could you keep the eori as parameter cause we need to return eori in EoriHistory (in case of failure) and then the EoriHistory would be used as part of AuthenticatedRequest in AuthAction? It will not log into Kibana.